data/catalogs/deltares_data.yml

```diff
@@ -1052,6 +1052,21 @@ grdc:
   rename:
     area: uparea
 
+grdc_csv:
```

Comment:
Perhaps better to name this one `grdc` and the excel version `grdc_excel`. If no issues arise (I don't expect any, but just to be sure), we can remove the excel version, but perhaps safer to keep this one for now as a fallback option.

Reply:
Yes that is better option. I might as well remove the excel version since the column info is also available at the source url.